<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QMainWindow" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>799</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_World"/>
    <addaction name="actionOpen_World"/>
    <addaction name="actionSave_World"/>
    <addaction name="actionSave_World_As"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionRun_Game"/>
   </widget>
   <widget class="QMenu" name="menuEntity">
    <property name="title">
     <string>Entity</string>
    </property>
    <addaction name="actionCreate"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Prefab_Library"/>
    <addaction name="actionLoad_Prefab_Library"/>
    <addaction name="actionSave_Prefab_Library"/>
    <addaction name="actionSave_Prefab_Library_As"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionEntities_List"/>
    <addaction name="actionComponents_List"/>
    <addaction name="actionComponent_Properties"/>
    <addaction name="actionPrefabs"/>
    <addaction name="actionTransform_Editor"/>
   </widget>
   <widget class="QMenu" name="menuWorld">
    <property name="title">
     <string>Components</string>
    </property>
    <addaction name="actionEdit_Components"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuEntity"/>
   <addaction name="menuWorld"/>
   <addaction name="menuWindow"/>
   <addaction name="menuRun"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_World">
   <property name="text">
    <string>Open World...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave_World">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save World</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_World_As">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Save World As...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+Z</string>
   </property>
  </action>
  <action name="actionNew_World">
   <property name="text">
    <string>New World</string>
   </property>
  </action>
  <action name="actionRun_Game">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionEntities_List">
   <property name="text">
    <string>Entities List</string>
   </property>
  </action>
  <action name="actionComponents_List">
   <property name="text">
    <string>Components List</string>
   </property>
  </action>
  <action name="actionComponent_Properties">
   <property name="text">
    <string>Component Attributes</string>
   </property>
  </action>
  <action name="actionLoad_Prefab_Library">
   <property name="text">
    <string>Open Prefab Library...</string>
   </property>
  </action>
  <action name="actionSave_Prefab_Library">
   <property name="text">
    <string>Save Prefab Library</string>
   </property>
  </action>
  <action name="actionSave_Prefab_Library_As">
   <property name="text">
    <string>Save Prefab Library As...</string>
   </property>
  </action>
  <action name="actionNew_Prefab_Library">
   <property name="text">
    <string>New Prefab Library</string>
   </property>
  </action>
  <action name="actionPrefabs">
   <property name="text">
    <string>Prefabs Library</string>
   </property>
  </action>
  <action name="actionTransform_Editor">
   <property name="text">
    <string>Transform Editor</string>
   </property>
  </action>
  <action name="actionCreate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </action>
  <action name="actionEdit_Components">
   <property name="text">
    <string>Edit World Components</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>mainwindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
